akka-cluster/src/main/scala/akka/cluster/ClusterDaemon.scala

```diff
@@ -1291,6 +1294,9 @@ private[cluster] class ClusterCoreDaemon(publisher: ActorRef) extends Actor with
       throw new IllegalStateException(s"Too many vector clock entries in gossip state ${latestGossip}")
 
   def publish(newGossip: Gossip): Unit = {
+    if (cluster.settings.Debug.VerboseGossipLogging)
```

Comment:
I think I made it _too_ verbose but maybe we can revisit that it prints on each publish later.